addons/medical/stringtable.xml

```diff
@@ -47,6 +47,7 @@
             <English>Epinephrine Wake Up Chance Boost</English>
             <Polish>Zwiększenie szansy na wybudzenie po podaniu epinefryny</Polish>
             <Chinese>腎上腺素甦醒率加成</Chinese>
+            <German>Epinephrin erhöht die Aufwachwahrscheinlichkeit</German>
```

Comment:
The phrasing of this string changed, and should be re-translated
the setting changed from true/false to a numerical value for amount of "boost"

Reply:
```suggestion
            <German>Erhöhung der Aufwachüberprüfungsrate durch Epinephrin</German>
```
Gods, this language is such a mess sometimes.

Reply:
Es geht aber noch viel schlimmer.